server/app/views/admin/programs/ProgramApplicationListView.java

```diff
@@ -357,10 +368,7 @@ private DivTag renderApplicationListItem(
             .condWith(
                 displayStatus,
                 p().withClasses("text-sm", "text-gray-700")
-                    .with(
-                        span("Status: "),
-                        span(application.getLatestStatus().orElse("None"))
-                            .withClass("font-semibold")))
+                    .with(span("Status: "), span(statusString).withClass("font-semibold")))
```

Comment:
**suggestion:** I changed the formatting for eligibility status to truncate the tag after the word as well as show green in the case that they may qualify. I think these statuses should follow a similar pattern (with green for when approved)?https://github.com/civiform/civiform/pull/4283/files

Reply:
Oh right!  I meant to change that after you made that change and completely forgot.  Thanks for the reminder!

Reply:
I'm not sure we can actually set green for approved, because it will be up to the admin to create an "approved" status to set it to, so we'd have to have a different mechanism to say that a particular status is the "approved" status.

Reply:
Yeah, I think we'll just leave it blue and maybe later on we can think about if we want to create some default "Approved, Rejected, Waiting" statuses or something.